Sources/Interaction/Style/InteractorStyleManipulator/index.js

```diff
@@ -686,6 +755,16 @@ function vtkInteractorStyleManipulator(publicAPI, model) {
       }
```

Comment:
use `renderer` instead of `model.getRenderer(callData)`

Reply:
yes.